test/simple.ll

```diff
@@ -146,8 +143,8 @@ attributes #2 = { nounwind readnone }
 !4 = !{!"int*", !"int*", !"int"}
```

Comment:
I guess these should be changed as well as !8 and !9 below

Reply:
Good find! This belongs to the kernel arg metadata so shouldn't be changed. And neither should `!8` and `!9` so I've taken this file out now.

I also found a few more, which I've also taken out.